presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -755,6 +755,40 @@ Procedures
     Unregisters given, existing partition in the metastore for the specified table.
     The partition data is not deleted.
 
+Special Columns
+---------------
+
+In addition to the defined columns, the Hive connector automatically exposes
+metadata in a number of hidden columns in each table. You can use these columns
+in your SQL statements like any other column, e.g., they can be selected
+directly or used in conditional statements.
+
+* ``$bucket``
+    Bucket number for this row.
+
+* ``$path``
+    Full file system path name of the file for this row.
+
+* ``file_modified_time``
+    Date and time of the last modification of the file for this row.
+
+* ``file_size``
+    Size of the file for this row.
+
+Table Properties
```

Comment:
Let's make this a new section so that we have a place to document other tables later (like the partitions table).
```
Special Tables
--------------

Table Properties
^^^^^^^^^^^^^^^^

Reply:
done